code/game/objects/structures/window.dm

```diff
@@ -511,16 +631,18 @@
 	icon = 'icons/obj/smooth_structures/reinforced_window.dmi'
 	icon_state = "r_window"
 	dir = FULLTILE_WINDOW_DIR
-	max_integrity = 100
+	max_integrity = 150
 	fulltile = TRUE
 	flags_1 = PREVENT_CLICK_UNDER_1
 	smooth = SMOOTH_TRUE
+	state = RWINDOW_SECURE
 	canSmoothWith = list(/obj/structure/window/fulltile, /obj/structure/window/reinforced/fulltile, /obj/structure/window/reinforced/tinted/fulltile, /obj/structure/window/plasma/fulltile, /obj/structure/window/plasma/reinforced/fulltile)
 	level = 3
 	glass_amount = 2
 
 /obj/structure/window/reinforced/fulltile/unanchored
 	anchored = FALSE
+	state = WINDOW_OUT_OF_FRAME
```

Comment:
Would it be possible to rename WINDOW_OUT_OF_FRAME to WINDOW_UNANCHORED, to avoid confusion with the new RWINDOW_OUT_OF_FRAME, which is 2 stages ahead of this stage?

Other than that couldn't find anything obviously fucked.

Reply:
out of frame isn't the same as unanchored, they're related but anchoring is actually the first construction step and it's separate.